cmd/zetae2e/local/local.go

```diff
@@ -246,6 +246,10 @@ func localE2ETest(cmd *cobra.Command, _ []string) {
 			e2etests.TestERC20DepositRestrictedName,
 		}
 		zetaTests := []string{
+			e2etests.TestMessagePassingZEVMToEVMName,
+			e2etests.TestMessagePassingEVMtoZEVMName,
+			e2etests.TestMessagePassingEVMtoZEVMRevertName,
+			e2etests.TestMessagePassingZEVMtoEVMRevertName,
```

Comment:
Can we put these below in `zetaAdvancedTests`?

This might be a bit confusing and maybe we should refactor but basically the advanced tests are skipped in migration test
Here it would fail because we would try to run the tests on the previous version, which would be not supported